app/src/main/java/com/cornellappdev/transit/ui/components/LocationItem.kt

```diff
@@ -0,0 +1,86 @@
+package com.cornellappdev.transit.ui.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.transit.R
+import com.cornellappdev.transit.ui.theme.TransitBlue
+
+@Composable
+fun LocationItem(image: Painter, editImage: Painter, label: String, sublabel: String, editing: Boolean){
+    Column(modifier =  Modifier.width(88.dp).height(86.dp).padding(horizontal = 8.dp)){
+        if(editing && label!="Add"){
```

Comment:
I can smell an unlinted file from a mile away 👀 

Make sure to run the formatter on your code:

![image](https://github.com/cuappdev/ithaca-transit-compose/assets/47724806/e879a176-2fe6-4d2b-b758-a5d460f59c9a)